File: swagger/models/contact_log_sub_type.py
from .base_model import BaseModel


class ContactLogSubType(BaseModel):
	"""
	A contact log subtype.

	Attributes:

		id
		The contact log subtype’s ID.
	"""

	input_map = {
		'Id': 'id',
		}

	output_map = {
		'id': 'Id',
		}

	id = None

File: swagger/Models/ContactLogSubType.php
<?php

namespace App\Mindbody\Models;

/**
 * A contact log subtype.
 */
class ContactLogSubType extends BaseModel
{
	/**
	 * The contact log subtype’s ID.
	 * 
	 * @var integer format:int32
	 */
	public $id;

	protected function getInputMap()
	{
		return [
			'Id' => ['id', 'integer', 'int32'],
			];
	}

	protected function getOutputMap()
	{
		return [
			'id' => ['Id', 'integer', 'int32'],
			];
	}
}
